packages/main/src/plugin/container-registry.ts

```diff
@@ -619,6 +620,12 @@ export class ContainerProviderRegistry {
       throw new Error('No provider with a running engine');
     }
 
+    // prefer podman over other engines
+    // sort by podman first as container type
+    matchingContainerProviders.sort((a, b) => {
+      return b[1].connection.type.localeCompare(a[1].connection.type);
```

Comment:
Why do you rely on the sorting and not take directly a podman connection using its `type === podman`? If in future an extension adds a new provider it could return an invalid value.

Reply:
I thought it was smarter

A connection is either using 'podman' or 'docker' type

so we sort using podman first

I can change

Reply:
I mean, it works fine. If you don't see a foreseeable issue for the future it can be merged like this for me. Just asking.

Reply:
@lstocchi PR updated